source/state_representation/src/space/cartesian/CartesianState.cpp

```diff
@@ -668,6 +673,8 @@ CartesianState& CartesianState::operator*=(const CartesianState& state) {
   Eigen::Vector3d f_omega_b = this->get_angular_velocity();
   Eigen::Vector3d f_a_b = this->get_linear_acceleration();
   Eigen::Vector3d f_alpha_b = this->get_angular_acceleration();
+  Eigen::Vector3d f_F_b = this->get_force();
+  Eigen::Vector3d f_T_b = this->get_torque();
```

Comment:
nitpick: Capital T is usually for transforms. for the angular one that uses greek letter the other ones are spelled out (e.g. omega above line 673). I recommend to use `f_tau_b` for consistency.

Reply:
Thanks, `tau` is better. But actually I had planned to refactor this operation to clean up these intermediate variables and make it generally more readable. A lot of the terms are only used once, while some of the other products are used multiple times (like `f_R_b * b_P_c`).

Reply:
Yes agreed. I followed the mathematical notation which, in fact, is not really readable without the proper material. If you can update that as well then even better.

Reply:
Are you still planning to clean those in this PR? If not then I would still suggest to replace `T` with `tau`.

Reply:
Thanks, I forgot. Because there is less stuff going on with the wrench, I did not clean this up further. With the addition of comments I think it's readable enough, and the rest should be optimized away by the compiler anyway.